security.txt

```diff
@@ -0,0 +1,9 @@
+Contact: mailto:info@friendi.ca
+
+Expires: Wed, 30 Jun 2021 23:59 +0000
```

Comment:
Is the expiry date always in the timeframe of the release?

Reply:
The goal is to update the expiration date each release, yes.